api/v4/source/access_control.yaml

```diff
@@ -538,6 +538,43 @@
           $ref: "#/components/responses/NotFound"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /api/v4/access_control_policies/sync_job:
+    post:
+      tags:
+        - access control
+      summary: Create an access control sync job
+      description: |
+        Creates a new access control sync job for a specific policy. Cancels any existing pending or in-progress sync jobs for the same policy.
+        ##### Permissions
+        Must have the `manage_system` permission.
```

Comment:
The API code says it needs "permission to create job", and here it says that it needs `manage_system` permissions. Maybe only sysadmins have "permission to create job", but wanted to double check, just in case.